eppo/src/main/java/cloud/eppo/android/ConfigurationStore.java

```diff
@@ -65,63 +66,64 @@ public void loadFromCache(CacheLoadCallback callback) {
                     return;
                 }
 
-                flags = configResponse.getFlags();
-                Log.d(TAG, "Cache loaded successfully");
+                flags = new ConcurrentHashMap<>(configResponse.getFlags());
```

Comment:
kudos for using `ConcurrentMap`

Reply:
It was there before I modified things so I can't take credit. I did make the higher-level abstractions just using `Map` and `ConcurrentHashMap` as the concrete implementation.